### PR TITLE
ESLintConfig

### PR DESCRIPTION
**Pull request by alexnewmannn**
EsLint config for bede.
Based on AirBnb's with a few additional rules

### CODE REVIEW

**Comment by pmcatominey** [comment]
🍧

File: linters/eslint-config-bede/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "type": "git",
+    "url": "git+https://github.com/BedeGaming/eslint-config-bede.git"
+  },
+  "keywords": [
+    "eslint"
+  ],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/BedeGaming/eslint-config-bede/issues"
```
Comment (mintbridge) [inline linters/eslint-config-bede/package.json]: urls?

Comment (alexnewmannn) [inline linters/eslint-config-bede/package.json]: ta

**Comment by mintbridge** [comment]
looks good, think we need to tidy the whole repo up though as well @ahumphreys87 ?

File: linters/eslint-config-bede/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "extends": [
+    'eslint-config-airbnb-base',
+    'eslint-config-airbnb-base/rules/strict',
+  ],
+  "rules": {
+    'max-len': [2, 120, 2, {
+      'ignoreUrls': true,
+      'ignoreComments': false
+    }],
+    "comma-dangle": [2, "never"],
```
Comment (ahumphreys87) [inline linters/eslint-config-bede/.eslintrc]: whats the `2` for?

Comment (alexnewmannn) [inline linters/eslint-config-bede/.eslintrc]: error level

Comment (alexnewmannn) [inline linters/eslint-config-bede/.eslintrc]: 0 wont throw anything, 1 warn, 2 error

File: linters/eslint-config-bede/.eslintrc [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "no-restricted-syntax": [0]
+  }
+}
```
Comment (ahumphreys87) [inline linters/eslint-config-bede/.eslintrc]: is this file needed given we have index.js?

Comment (alexnewmannn) [inline linters/eslint-config-bede/.eslintrc]: nope, ive actually added it to `linters` forgot to remove it here

**Comment by ahumphreys87** [comment]
:shipit:
